flink-runtime/src/main/java/org/apache/flink/runtime/checkpoint/CompletedCheckpoint.java

```diff
@@ -281,6 +285,34 @@ private void doDiscard() throws Exception {
 		}
 	}
 
+	private Set<KeyedStateHandle> getIncrementKeyedStates() {
```

Comment:
The real target you want is actually `getSharedStates`

Reply:
yes，do you a better suggestion？

Reply:
I think I have offered the method name as `getSharedStates`